### ISSUE TITLE
'High Contrast White' theme Avatar color contrast

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1029]
**Describe the bug**
Icons in the Avatar component do not have a contrast ratio of 3:1 or above.
Icons that can be used to understand the content need a minimum contrast ratio of 3:1 according to this WCAG 2.1 specification: https://www.w3.org/WAI/WCAG21/Techniques/general/G207.html 

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/avatar.html
2. Check Icon color contrast

**Expected behavior**
Icon in the Avatar component should have a color contrast ratio of 3:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81855034-9a11da80-9513-11ea-80bd-75adb1d09bf7.png)
![image](https://user-images.githubusercontent.com/43764832/81855063-a433d900-9513-11ea-8f12-b2601071c760.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 5.3:1
<img width="377" alt="Screen Shot 2020-10-07 at 5 38 27 PM" src="https://user-images.githubusercontent.com/21978807/95529883-15779b00-0991-11eb-893c-fd5d49d24f01.png">

**Comment by prsdthkr** [comment]
See this page in HCW Theme:
https://deploy-preview-1752--fundamental-styles.netlify.app/?path=/docs/components-avatar--icon
